shared/team-building/phone-search.tsx

```diff
@@ -86,6 +86,7 @@ const PhoneSearch = (props: PhoneSearchProps) => {
             key={phoneInputKey}
             autoFocus={true}
             onChangeNumber={onChangeNumberCb}
+            onEnterKeyDown={_onContinue}
```

Comment:
The continue button is disabled on `!validity`, maybe this callback can bail in that case

Reply:
@buoyad Great catch, thanks!